apps/webapp/package.json

```diff
@@ -6,8 +6,8 @@
     "@lexical/history": "0.14.2",
     "@lexical/react": "0.14.2",
     "@wireapp/avs": "9.6.12",
-    "@wireapp/commons": "5.2.7",
-    "@wireapp/core": "45.3.0",
+    "@wireapp/commons": "workspace:^",
+    "@wireapp/core": "workspace:^",
```

Comment:
WE now depend on the package version in the workspace, no more yalcing needed anymore 😎